charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -36,16 +36,6 @@ victoria-metrics-operator:
     # -- By default, operator converts prometheus-operator objects.
     disable_prometheus_converter: false
 
-serviceAccount:
```

Comment:
Would you mind mentioning in the changelog that we removed this? Even though it didn't do anything, it might still be used by some users.

Reply:
added this here https://github.com/VictoriaMetrics/helm-charts/pull/1662